ArchiSteamFarm/Bot.cs

```diff
@@ -439,16 +439,34 @@ internal sealed class Bot {
 		}
 
 		internal static string ResponseStatus(string botName) {
+			Bot bot;
+			string result="";
 			if (string.IsNullOrEmpty(botName)) {
 				return null;
 			}
 
-			Bot bot;
 			if (!Bots.TryGetValue(botName, out bot)) {
-				return "Couldn't find any bot named " + botName + "!";
+				result+="Couldn't find any bot named " + botName + "!";
+				return result;
+			}
+			
+			if (bot.CardsFarmer.CurrentGamesFarming.Count > 0) {
+				result+="Bot " + bot.BotName + " is currently farming appIDs: " + string.Join(", ", bot.CardsFarmer.CurrentGamesFarming) + " and has a total of " + bot.CardsFarmer.GamesToFarm.Count + " games left to farm.\n";
 			}
+			result+="Currently " + Bots.Count + " bots are running";
+			return result;
+		}
 
-			return "Bot " + bot.BotName + " is currently farming appIDs: " + string.Join(", ", bot.CardsFarmer.CurrentGamesFarming) + " and has a total of " + bot.CardsFarmer.GamesToFarm.Count + " games left to farm";
+		internal static string ResponseStatusAll() {
+			string result = "";
+			foreach (var curbot in Bots) {
+				if (curbot.Value.CardsFarmer.CurrentGamesFarming.Count == 0)
+					result += "Bot " + curbot.Key + " is not farming.\n";
```

Comment:
Unsure if this is good idea, but I made it. Actually, the environment where asf running, and the steam client or browser where answer is showed - is too different things. We need to check it, but I think any receiving side would be satisfied with \n